package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pretest": "eslint --rulesdir=tools/eslint-rules lib test",
-    "test": "tap \"test/**/*.test.js\"",
+    "test": "tap \"test/cli/*.test.js\"",
```

Comment:
This glob is interpreted differently, depending on who is running our test suite. This means that if we ever add tests outside of `test/cli` we'll have to change this.